### PR TITLE
Globbing patterns for options.includes

### PR DESCRIPTION
**Pull request by AdamQuadmon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ ghost: {
       direct: true,
       // Sets logging level, check out http://casperjs.org/logging.html
       logLevel: 'info',
-      // Specifies files to be included for each test file
+      // Specifies files to be included for each test file, now accepts globbing patterns
```
Comment (colinwren) [inline README.md]: This should be 

```
accepts file paths or globs as input
```

File: tasks/ghost.js [hunk 2]
```diff
@@ -29,7 +29,14 @@ module.exports = function (grunt) {
 
     if (options.direct) command.push('--direct');
 
-    if (options.includes) command.push('--includes=' + options.includes.join());
+    if (options.includes) {
+      // Get filepaths from 'options.includes'
+      var includes = [].concat(
```
Comment (colinwren) [inline tasks/ghost.js]: Why are you using `[].concat(grunt.file.expand(options.includes))` instead of just grunt.file.expand(options.includes)?

**Comment by colinwren** [comment]
This looks great! I commented on something I was unclear about, lets resolve those and then i'll merge this!

**Comment by AdamQuadmon** [comment]
thanks for the tips, look now!

**Comment by colinwren** [comment]
Thanks for your contribution!

**Comment by AdamQuadmon** [comment]
thank you for this task!

**Pull request merged**